src/user/user.controller.ts

```diff
@@ -0,0 +1,55 @@
+import { Router, Request, Response, NextFunction } from 'express';
+import { IUser } from './user.interface';
```

Comment:
Consider grouping imports of the same level together (`./oneThing` and `../anotherThing` are different levels) and separating each level with an empty line. Additionally, order imports top-down from most-global to most-local.
E.g. https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md

If you've got time, consider adding this plugin to your project to automate that.